debian/jitsi-meet-web-config.postinst

```diff
@@ -182,36 +186,57 @@ case "$1" in
             echo "config.flags.receiveMultipleVideoStreams = true;" >> $JITSI_MEET_CONFIG
         fi
 
-        if [[ "$FORCE_NGINX" = "true" && ( -z "$JVB_HOSTNAME_OLD" || "$RECONFIGURING" = "true" ) ]] ; then
+        NGINX_CONF_PATH="/etc/nginx/sites-enabled/$JVB_HOSTNAME_OLD.conf"
+        OPENRESTY_CONF_PATH="/etc/nginx/sites-enabled/$JVB_HOSTNAME_OLD.conf"
+
+        if [[ "$FORCE_OPENRESTY" = "true" ]]; then
+            NGX_COMMON_CONF_PATH="$OPENRESTY_CONF_PATH"
+            NGX_SVC_NAME=openresty
+            OPENRESTY_NGX_CONF="/usr/local/openresty/nginx/conf/nginx.conf"
+        else
+            NGX_COMMON_CONF_PATH="$NGINX_CONF_PATH"
+            NGX_SVC_NAME=nginx
+        fi
+
+        if [[ ( "$FORCE_NGINX" = "true" || "$FORCE_OPENRESTY" = "true" ) && ( -z "$JVB_HOSTNAME_OLD" || "$RECONFIGURING" = "true" ) ]] ; then
 
             # this is a reconfigure, lets just delete old links
             if [ "$RECONFIGURING" = "true" ] ; then
-                rm -f /etc/nginx/sites-enabled/$JVB_HOSTNAME_OLD.conf
+                rm -f "$NGINX_CONF_PATH"
+                rm -f "$OPENRESTY_CONF_PATH"
```

Comment:
Why deleting twice the same file?

Reply:
It is all right in cotext of right values of variables (I've already pushed the fix. Honestly, I don't know how on the earth I pushed this values, as I was pretty sure that I set proper values there :shrug: )

Reply:
although, wait.
I just noticed, that exactly here it is not right.

Here is `enabled`, and NgX variable is intended to be (and, now it is, after I pushed fix) `*-available`.

So, this block removes only "enabled" link (what's the point, btw?).

And there is no "enabled" thing for openresty (I just put config itself there).

So, I'll remove openresty from here, I think...